.ci/Jenkinsfile-compile

```diff
@@ -43,7 +43,7 @@ pipeline {
           ]
 
           def nuttx_builds_other = [
-            target: ["px4_cannode-v1_default", "px4_esc-v1_default", "thiemar_s2740vc-v1_default"],
+            target: ["px4_esc-v1_default", "thiemar_s2740vc-v1_default"],
```

Comment:
No more `cannode` whatever that is? :smile:

Reply:
There's a weird circular dependency issue I won't get into. Suffice it to say, it'll be easier to gradually untangle the "px4 platform" dependencies, then bring this (unused) target back later.